Update interfaces/css-typed-om.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/w3c/webref/blob/773da44/ed/idl/css-typed-om.idl
Build: https://travis-ci.org/w3c/webref/builds/214263241

foolip commented on interfaces/css-typed-om.idl:
```diff
@@ -316,3 +316,75 @@ dictionary CSSMatrixComponentOptions {
+
+[Exposed=(Window, Worker, PaintWorklet, LayoutWorklet)]
+interface CSSColor : CSSColorValue {
+    constructor(sequence<DOMString or CSSNumberish> variant)
```
There's a missing semicolon here breaking the test.

foolip replied:
I've sent https://github.com/w3c/css-houdini-drafts/pull/1017 to add the semicolon.

foolip replied:
Nope https://github.com/w3c/css-houdini-drafts/pull/1016 is it!

autofoolip committed:
Update interfaces/css-typed-om.idl

Source: https://github.com/w3c/webref/blob/773da44/ed/idl/css-typed-om.idl
Build: https://travis-ci.org/w3c/webref/builds/214263241
